README.md

```diff
@@ -243,6 +244,7 @@ like function parameters or arguments.
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   textobjects = {
+    enable = true, 
```

Comment:
This isn't needed, the textobject itself is not a module, it has submodules that needs to be activated one by one.